### ISSUE TITLE
SQL sp_password in comment but should not be flagged

### ISSUE THREAD

**Issue by hchiam** [hchiam/custom-vscode-linter#21]
Should not be flagging this:
```sql
AS--WITH ENCRYPTION

BEGIN

  -- comment: sp_password
```
